actionpack/test/dispatch/routing/inspector_test.rb

```diff
@@ -305,6 +305,16 @@ def test_routes_can_be_filtered
                       "          DELETE /posts/:id(.:format)      posts#destroy"], output
       end
 
+      def test_relative_url_root
+        Rails.application.config.stubs(:relative_url_root).returns('/foo')
```

Comment:
We're trying to phase mocha out of our test suite. Please use the stub method from minitest instead.